app/views/requests_interests.py

```diff
@@ -0,0 +1,35 @@
+from django.http import Http404
+from rest_framework import generics
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from ..models import RequestInterest
+from ..serializers import RequestInterestSerializer
+
+
+class RequestInterestList(generics.ListCreateAPIView):
+    permission_classes = (IsAuthenticated,)
+    queryset = RequestInterest.objects.all()
+    serializer_class = RequestInterestSerializer
+
+
+class MentorRequestInterest(APIView):
+    permission_classes = (IsAuthenticated,)
+
+    def get_object(self, pk):
```

Comment:
I don't know if `get_object` is a good name here, being that we're getting a list of interests